NIMATRO/NIMATRO-IOC-01App/Db/controls.db

```diff
@@ -0,0 +1,422 @@
+record(bo, "$(P)SIM")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM=0)")
+}
+
+record(bo, "$(P)DISABLE") 
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE=0)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+
+# Initially generated by $Id$
+# 
+# auto-generated EPICS records specify an asyn port "lvfp", but this can be changed - it just needs
+# to match the first argument of the relevant lvDCOMConfigure() command in the IOC st.cmd 
+# 
+      
+# 
+# Definitions from configSection "frontpanel" in XML input file
+#
+#     lvDCOMConfigure("lvfp", "frontpanel", "/path/to/lvinput.xml")
+# 
+
+# Write to LabVIEW control "Speed" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+record(ai, "$(P)SPEED")
+{
+	field(DESC, "LabVIEW 'Speed'")
+    field(DTYP, "asynFloat64")
+	field(SCAN, ".5 second")
+    field(INP,  "@asyn(lvfp,0,0)Speed")
+    field(PREC, "3")
+    field(EGU, "cm^2/min")
+    info(INTEREST, "HIGH")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:SPEED")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ao, "$(P)SPEED:SP")
+{
+	field(DESC, "LabVIEW 'Speed'")
+    field(DTYP, "asynFloat64")
+    field(OUT,  "@asyn(lvfp,0,0)Speed")
+    field(PREC, "3")
+    field(EGU, "cm^2/min")
+	field(DRVH, "174.2")
+	field(DRVL, "-174.2")
```

Comment:
A quick comment explaining this number would be helpful - even if it just says "this is a hardware limit". Otherwise it just looks like a magic number

Reply:
Added comments to clarify it's device derived